check.py

```diff
@@ -360,6 +360,10 @@ def run_binaryen_js_tests():
     f = open('a.js', 'w')
     binaryen_js = open(os.path.join(options.binaryen_bin, 'binaryen.js')).read()
     f.write(binaryen_js)
+    # running concatenated versions in node interferes with module loading because
```

Comment:
there is similar code in `auto_update_tests.py`.

we need to verify that code works in both spidermonkey and node.js. the bots test just one.

Reply:
Assumption here is that, in spidermonkey, there isn't a global `module` or `exports` so that the last case of the module loader sets up a global, just like in a browser, and the workaround is skipped (condition is `false`).

```js
else
  (typeof self !== "undefined" ? self : this)['Binaryen'] = instantiate();
```

But I agree, that should also be tested.